store/src/java/com/zimbra/cs/account/soap/SoapDistributionList.java

```diff
@@ -81,6 +81,13 @@
                 SoapProvisioning.getAttrs(e), prov);
         addDlm(e, getRawAttrs());
     }
+    
+    //ZBUG-651: This bug is about to get correct member details for gadl command;
+  	//ZBUG-651: Below constructor is added, which does not call addDlm() as this method is not allowing to get member details
+  	//ZBUG-651: Added one more boolean parameter in here, to differentiate from other constructor
+  	SoapDistributionList(DistributionListInfo dlInfo, Provisioning prov, boolean hideMembers) throws ServiceException {
```

Comment:
from where exactly we are calling this constructor form? most of the callers are using two params when creating instance, there is no caller which is using three params in object instantiation

Reply:
Added the missed out changes into the PR. The changes were missed while squashing process.

Reply:
on line no 62, we have a constructor which should ideally set member data properly, isn't working or not called at all?

Reply:
Before calling addDlm(dlInfo.getMembers(), getRawAttrs()) method, we are getting members value. And in this method, we further calling getMembers() in which member value is not setting from anywhere hence it is zero. And inside addDlm(), we are putting member value in map and that's where member value gets override to zero.